### PR TITLE
batch-sign assemblies using the Roslyn SignTool

### PR DESCRIPTION
**Pull request by brettfo**
This moves assembly signing to one large batch process after all assemblies have been built.  This should reduce the time of a fully signed internal build from 4 hours to just over 1.

Initial signed build completed in 80 minutes.  I'll now prepare a build for a test insertion into Visual Studio.

Special thanks to @jaredpar for helping me with this.

### CODE REVIEW

**Comment by brettfo** [comment]
@dotnet-bot test Ubuntu14.04 Release_fcs Build please

**Comment by brettfo** [comment]
@dotnet-bot test Windows_NT Release_fcs Build please

**Comment by brettfo** [comment]
@dotnet-bot test Windows_NT Release_fcs Build please

File: build/config/SignToolData.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        "msi\\Microsoft.FSharp.SDK.Core.msi",
+        "msi\\Microsoft.FSharp.SDK.Resources.*.msi"
+      ]
+    },
+    {
+      "certificate": "MicrosoftSHA2",
+      "strongName": null,
+      "comment": "TODO: test binaries go here",
+      "values": [
+      ]
```
Comment (jaredpar) [inline build/config/SignToolData.json]: Think you can remove this section.

**Comment by brettfo** [comment]
All internal sign-checks pass.  I'm doing one final validation build to ensure the assemblies load at runtime.

**Pull request merged**